samples/sensor/apds9960/README.rst

```diff
@@ -1,52 +1,38 @@
-.. _apds9960-sample:
+.. _apds9960:
 
-RGB and Gesture Sensor
-######################
+APDS9960 RGB, ambient, Gesture Sensor
+#####################################
 
 Overview
 ********
 
-This sample utilizes APDS-9960 Sensor and reads RGB values from the sensor
-then displays the color through the APA102C LED.
+A sensor application that demonstrates how to use the APDS9960 to get ambient light,
+RGB and proximity data. This sample only checks sensor in polling mode(without trigger)
```

Comment:
add a space before the parenthesis, and a final period. How about saying ...  "poling mode (without an interrupt trigger)."